addons/l10n_vn/models/res_partner_bank.py

```diff
@@ -0,0 +1,44 @@
+#  Copyright (c) by The Bean Family, 2023.
+#
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+#  These code are maintained by The Bean Family.
+import json
+from email.policy import default
+from odoo import models, fields, api, _
+
+
+""" Customize the res_partner_bank model
+    - Add compute function for @currency_id to get the default currency_id of current Company
+    - Override the original 'retrieve_acc_type' function to add more bank account type.
+"""
+
+class ResPartnerBank(models.Model):
+    _inherit = "res.partner.bank"
+
+    currency_id = fields.Many2one('res.currency', string='Currency', default=lambda self: self.env.company.currency_id)
```

Comment:
I don't think we would want that, even less only if you have `l10n_vn` installed.

Reply:
After review about the use case for this field, i agree with you. I will remove this field for my PR. 

Thank you for your comment